int32 roomId
string roomName
float32 batteryPercentage